MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+# Exclude pyproject.toml to avoid using a PEP 517 backend:
+# - https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support
+# - https://www.python.org/dev/peps/pep-0517
+exclude pyproject.toml
```

Comment:
`sqlalchemy/alembic` does the same thing to avoid including `pyproject.toml`: https://github.com/sqlalchemy/alembic/commit/5be6b9c1517ece3637e0a04a44479c7daf9f63d9